src/main/java/io/javalin/http/Context.kt

```diff
@@ -137,6 +138,7 @@ open class Context(@JvmField val req: HttpServletRequest, @JvmField val res: Htt
     fun <T> bodyValidator(clazz: Class<T>) = try {
         Validator(JavalinJson.fromJson(body(), clazz), "Request body as ${clazz.simpleName}")
     } catch (e: Exception) {
+        Javalin.log?.error("Couldn't deserialize body to ${clazz.simpleName}", e);
```

Comment:
`error` seems a bit extreme. If a client provides a bad object, your logs shouldn't be littered with error messages (might be nothing wrong with your app). How about `info` ?

Reply:
If I understand the flow correctly this code is only invoked when you are attempting to serialize an existing object to json instead of trying to deserialize user input.  This situation seems like one warranting throwing error messages everywhere.

Reply:
You make a good point about not wanting error messages littering the logs from bad client objects. I think info suffers from the same problem though, in that it is a default level choosen. Maybe debug would be a better choice as it makes sense to enable debug level logging to figure out why something isnt working.

Reply:
Debug is even better!